src/main/java/TaskList.java

```diff
@@ -0,0 +1,179 @@
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Represents the task list handler.
+ */
+public class TaskList {
+    private static final List<Task> storage = new ArrayList<>();
```

Comment:
Perhaps you can use a better constant name (i.e. TASKS) and use all uppercase here?